app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>This is for all users</h1>
\ No newline at end of file
+<h1 class="post_heading">This is for all users</h1>
+
+<div id="users">
+  <% @users.each do |user| %>
+    <%= render partial: "shared/single_user", locals: { user: user } %>
+  <% end %>
+</div>
```

Comment:
* Kindly use `#recent_post` method to render the user's 3 most recent posts of a user, you have this methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to render the user's 3 most recent posts. [as mentioned in the project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.)